Documentation/ReleaseNotes.md

```diff
@@ -277,6 +260,17 @@ To work around this issue, please perform one of the following steps:
 - Switch the scripting backend to IL2CPP
 - In the Build Settings window, uncheck **Unity C# Projects"
 
+**Hang when using Holographic Remoting**
+
+There is a known issue with some versions of Unity where the editor may hang upon entering play mode during a remoting session. This issue may manifest if the Holographic window is open when the project is loaded.
+
+To work around the issue, please perform the following steps:
+
+1. With the project open, close the Holographic dialog.
+1. Close Unity
+1. Reopen Unity and open the project.
+1. Ensure that the Holographic dialog is closed before re-closing Unity.
```

Comment:
If I'm reading this bullet point correctly, does this mean that I'm closing Unity and reopening it twice?

Or is this last bullet point saying "make sure you actually RTFM for step 1"?

Reply:
It is just a reminder to close the dialog when complete. i can see how it reads like "be sure to do step 1"... i'll remove the last step

Reply:
removed